Week-11/lightswitch/src/reducers/changeState.js

```diff
@@ -0,0 +1,8 @@
+const changeState = (state=false,action) => {
+    switch(action.type){
+        case 'change': return state = action.payload;

```

Comment:
All the actions should be in a file at one place. Don't hardcode them because in future you may unable to track the names.